### PR TITLE
Correct PCIe 3.0 speed

### PR DESCRIPTION
**Pull request by scott4000**

### CODE REVIEW

File: documentation/asciidoc/computers/raspberry-pi/pcie.adoc [hunk 1]
```diff
@@ -55,7 +55,7 @@ WARNING: The Raspberry Pi 5 is not certified for Gen 3.0 speeds. PCIe Gen 3.0 co
 
 ==== via `config.txt`
 
-The connection is certified for Gen 2.0 speeds (5 GT/sec), but you can force Gen 3.0 (10 GT/sec) speeds. To enable PCIe Gen 3.0 speeds, add the following line to `/boot/firmware/config.txt`:
+The connection is certified for Gen 2.0 speeds (5 GT/sec), but you can force Gen 3.0 (8 GT/sec) speeds. To enable PCIe Gen 3.0 speeds, add the following line to `/boot/firmware/config.txt`:
```
Comment (nathan-contino) [inline documentation/asciidoc/computers/raspberry-pi/pcie.adoc:58]:
```suggestion
The connection is certified for Gen 2.0 speeds (5 GT/s), but you can force Gen 3.0 (8 GT/s) speeds. To enable PCIe Gen 3.0 speeds, add the following line to `/boot/firmware/config.txt`:
```
Suggestion: align this with our style guide (not your fault, just sprucing this up as well!)

**Review by nathan-contino — APPROVED** [review]
LGTM. Thank you for noticing and correcting this -- I don't know how we managed to mess this up originally.

File: documentation/asciidoc/computers/raspberry-pi/pcie.adoc [hunk 2]
```diff
@@ -55,7 +55,7 @@ WARNING: The Raspberry Pi 5 is not certified for Gen 3.0 speeds. PCIe Gen 3.0 co
 
 ==== via `config.txt`
 
-The connection is certified for Gen 2.0 speeds (5 GT/sec), but you can force Gen 3.0 (10 GT/sec) speeds. To enable PCIe Gen 3.0 speeds, add the following line to `/boot/firmware/config.txt`:
+The connection is certified for Gen 2.0 speeds (5 GT/s), but you can force Gen 3.0 (8 GT/s) speeds. To enable PCIe Gen 3.0 speeds, add the following line to `/boot/firmware/config.txt`:
```
Comment (lurch) [inline documentation/asciidoc/computers/raspberry-pi/pcie.adoc:58]: I guess the **first** sentence here (talking about transfer speeds) is equally-applicable to both "via `config.txt`" and "via `raspi-config`" so should probably be moved to before the header?

Comment (nathan-contino) [reply documentation/asciidoc/computers/raspberry-pi/pcie.adoc:58]: Actually, this is a perfect opportunity to use tabs. I'll get this fixed shortly with a new flow that uses them.

Comment (nathan-contino) [reply documentation/asciidoc/computers/raspberry-pi/pcie.adoc:58]: Fixed in https://github.com/raspberrypi/documentation/pull/3872

**Pull request merged**